docs/rollup.config.js

```diff
@@ -0,0 +1,34 @@
+const path = require('path');
+const copy = require('rollup-plugin-copy');
+const { createMpaConfig } = require('./_building-rollup/createMpaConfig.js');
+
+module.exports = async () => {
+  const mpaConfig = await createMpaConfig({
+    outputDir: '_site',
+    legacyBuild: true,
+    inputGlob: '_site-dev/**/*.html',
+    rootPath: path.resolve('_site-dev'),
+
+    // development mode creates a non-minified build for debugging or development
+    developmentMode: false, // process.env.ROLLUP_WATCH === 'true',
+
+    // set to true to inject the service worker registration into your index.html
+    injectServiceWorker: false,
```

Comment:
Being we've set this up, what's between here and turning the service worker back on?

Reply:
we need a good (non intrusive) way to notify users about new versions and/or load new version on menu interactions or so...

initially I had this on... but it triggered some frustrating situations where my update was actually correct I could just not see it without manually clearing the "app" cache...
so before we enable this we need to make it bullet prove

Reply:
that shouldnt even be a problem with MPAs, because its no longer a single page app, on every navigation, a new check for a service worker update is done. When a new service worker is found, we can listen for the controllerchange event and reload the page

Reply:
Makes sense to wait, if there is any question, but I _am_ excited to see coverage for this lingering on the horizon!

Reply:
I can dive into this this weekend, but imo its not blocking to merge this